docs/rfcs/0002/readme.md

```diff
@@ -37,21 +34,58 @@ The proposal below might take a couple of read-throughs; I've also added a concr
+    3. the NNF storage requirements described above.
+3. The ADMINISTRATOR creates a corresponding _NNF Container Profile_ Kubernetes custom resource with the necessary NNF storage requirements and pod specification as described by the AUTHOR
+4. The USER who desires to use the application works with the AUTHOR and the related NNF Container Profile to understand the storage requirements
+5. The USER submits a WLM job with the #DW container directive variables populated
+6. WLM runs the workflow and drives it through the following stages...
+    1. `Proposal`: RABBIT validates the #DW container directive by comparing the supplied values to those listed in the NNF Container Profile. If the workflow fails to meet the requirements, the job fails
+    2. `PreRun`: RABBIT software:
+        1. creates a config map reflecting the storage requirements and any runtime parameters; this is provided to the container at the volume mount named `nnf-config`, if specified
+        2. duplicates the pod template specification from the Container Profile and patches the necessary Volumes and the config map. The spec is used as the basis for starting the necessary pods and containers.
+    3. The containerized application(s) executes. The expected mounts are available per the requirements and celebration occurs. The pods continue to run until:
+       1. the pod completes successfully (any failed pods will be restarted)
+       2. timeout (i.e. `activeDeadlineSeconds`) is hit (optional)
+       3. the max number of pod retries (i.e. `backoffLimit`) is hit (indicating failure on all retry attempts)
+          1. Note: retry limit is non-optional per Kubernetes configuration
+          2. If retries are not desired, this number could be set to disable any retry attempts.
+    4. `PostRun`: RABBIT software:
+       1. Mark the stage as `Ready` if the pods have all completed successfully. This includes any retries after preceding failures.
+       2. If all pods are completed but did complete successfully, the stage is not be marked as ready.
+       3. Leave all pods around for log inspection
+
+### Communication Details
+
+Other than mounts, the following subsections outline the proposed communication between the Rabbit nodes themselves and the Compute nodes.
+
+#### Rabbit-to-Rabbit Communication
+
+A headless Kubernetes service connects the pods. This service is unique to each container workflow. Each rabbit node would be
+reached via `<host-name>.<service-name>`. The service-name would be provided to the application via a well-known environmental variable.
+This has been prototyped and has proven to be successful. The list of Rabbits available for communication is provided as well.
```

Comment:
You should include a concrete example

Reply:
What would you like to see here?

Reply:
The environmental variable and a sample config map